setup.py

```diff
@@ -1,16 +1,20 @@
 from setuptools import setup, find_packages
 
+
 def readme():
-    with open('README.rst') as f:
+    with open("README.rst") as f:
         return f.read()
 
-setup(name='protopipe',
-      version='0.1',
-      description='Pipeline to process events from DL0 to DL3',
-      url='http://github.com/jjlk/protopipe',
-      author='CEA',
-      author_email='julien.lefaucheur@cea.fr',
-      license='MIT',
-      packages=find_packages(),
-      install_requires=['ctapipe'],
-      zip_safe=False)
\ No newline at end of file
+
+setup(
+    name="protopipe",
+    version="0.2.1-dev",
```

Comment:
Error-prone when the version changes and you have several places to update.
You might prefer to read the version directly:
```
import protopipe
setup(
     name="protopipe",
     version=protopipe.__version__
```

Reply:
Yes, I agree!

Reply:
Is it costumary to import the package in here?
Like `import protopipe` and then `protopipe.__version__`?

Reply:
There are several ways to do it and I am not sure there is really one customary way.
You may also use
```
from pkg_resources import get_distribution
get_distribution('protopipe').version
```

Reply:
This way I would need to add a new dependence.
What if I add a `_version.py` file and do `from protopipe._version import __version__`?

Reply:
Then why not importing protopipe as suggested first?
It is customary to have the version in `__init__.py`.

Reply:
Yes, in the end, is the same, `__version__` gets imported from `__init__.py`